src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int maxCount = 3;
```

Comment:
1. please remember about NAMING_CONVENTION_FOR_CONSTANTS
2. Naming is not informative. Max count of what? We cannot obtain less than this max according to logic below. Please suggest something like BALLS_NUMBER etc.